config/crd/bases/networking.x-k8s.io_httproutes.yaml

```diff
@@ -71,6 +71,11 @@ spec:
                       description: GatewayReference identifies a Gateway in a specified
                         namespace.
                       properties:
+                        controller:
+                          description: Controller is the name of the controller implementing
+                            the Gateway.
+                          maxLength: 253
+                          type: string
```

Comment:
Is this an intended side-effect?
This would require Route owners to also define the controller and not just the Gateway. It seems that we are breaking our abstraction layers here.

Reply:
Good catch, I'd missed that addition here. I'd already moved on in my head to where we had a selector here and not a list. This is an optional field, but even still, it doesn't make sense here. I'll pull it out for this use case until we change the handshake entirely in v1alpha2.

Reply:
Created a new `GatewayStatusReference` to differentiate these two references.